### PR TITLE
Fixed dh issue by modifying get_bbox()

### PR DESCRIPTION
**Pull request by khawajasim**
Fixes issue #166

### CODE REVIEW

File: csep/core/forecasts.py [hunk 1]
```diff
@@ -437,7 +437,7 @@ def plot(self, ax=None, show=False, log=True, extent=None, set_global=False, plo
             axes: matplotlib.Axes.axes
         """
         # no mutable function arguments
-        dh = round(self.region.dh, 5)
+        # dh = round(self.region.dh, 5)
```
Comment (wsavran) [inline csep/core/forecasts.py:440]: can you remove these comments?

File: csep/utils/plots.py [hunk 2]
```diff
@@ -862,7 +862,7 @@ def plot_spatial_dataset(gridded, region, ax=None, show=False, extent=None, set_
     # Get spatial information for plotting
     bbox = region.get_bbox()
     if extent is None and not set_global:
-        extent = [bbox[0], bbox[1], bbox[2] + region.dh, bbox[3] + region.dh]
+        extent = [bbox[0], bbox[1], bbox[2], bbox[3]] # extent = [bbox[0], bbox[1], bbox[2] + region.dh, bbox[3] + region.dh]
```
Comment (wsavran) [inline csep/utils/plots.py:865]: unnecessary comment here as well

File: csep/utils/plots.py [hunk 3]
```diff
@@ -930,9 +930,9 @@ def plot_spatial_dataset(gridded, region, ax=None, show=False, extent=None, set_
     cmap = matplotlib.colors.ListedColormap(cmap_tup)
 
     ## Plot spatial dataset
-    lons, lats = numpy.meshgrid(numpy.append(region.xs, region.xs[-1] + region.dh),
-                                numpy.append(region.ys, region.ys[-1] + region.dh))
-
+    # lons, lats = numpy.meshgrid(numpy.append(region.xs, region.xs[-1] + region.dh),
+    #                             numpy.append(region.ys, region.ys[-1] + region.dh))
```
Comment (wsavran) [inline csep/utils/plots.py:934]: here as well

**Comment by wsavran** [comment]
@khawajasim looks good! i made a couple comments about some comments. other than that it looks good to go! this pr fixes #166

File: csep/core/forecasts.py (hunk 1 above)
Comment (khawajasim) [reply csep/core/forecasts.py:440]: Yea removing now.

**Comment by khawajasim** [comment]
Thanks for the merge. When is it going into the master branch now? 

I would like to add a few more functions in the coming days, mainly related to forecast mapping from one grid to another. Although, those functions will not be part of the QuadtreeGrid2D class. But generally part of region.m file. I want to see this through first and then add further.

Cheers,

**Pull request merged**